sandbox/euc101/main.tf

```diff
@@ -1,41 +1,297 @@
-################
-# AWS Provider #
-################
-provider "aws" {
-  region = var.region
-}
-
-########################
-# Configure S3 backend #
-########################
-terraform {
-  backend "s3" {
-    bucket         = "euc101-sandbox-terraform-state"
-    key            = "terraform.tfstate"
-    region         = "eu-central-1"
-    dynamodb_table = "tf_lock"
-  }
-}
-
-##############################################################
-# Common composed values shared across the different modules #
-##############################################################
-locals {
-  env_name = terraform.workspace
-  common_tags = {
-    EnvClass    = var.env_class
-    Environment = local.env_name
-    Owner       = "DevOps"
-    Terraform   = "true"
-  }
-}
-
-module "rabbitmq_security_group" {
-  source              = "terraform-aws-modules/security-group/aws//modules/rabbitmq"
-  version             = "~> 4.0"
-  create              = var.rabbitmq_create[local.env_name]
-  vpc_id              = var.vpc_id[local.env_name]
-  name                = "${local.env_name}-${var.env_class}-rabbitmq-security-group"
-  ingress_cidr_blocks = ["0.0.0.0/0"]
-  tags                = local.common_tags
-}
+############## AWS Provider ############
+provider "aws" {
+  region = var.region
+}
+
+########## Configure S3 backend #########
+terraform {
+  backend "s3" {
+    bucket         = "euc101-sandbox-terraform-state"
+    key            = "terraform.tfstate"
+    region         = "eu-central-1"
+    dynamodb_table = "tf_lock"
+  }
+}
+
+############ Common composed values shared across the different modules ############
+locals {
+  env_name = terraform.workspace
+  common_tags = {
+    EnvClass    = var.env_class
+    Environment = local.env_name
+    Owner       = "DevOps"
+    Terraform   = "true"
+  }
+}
+
+########## Rassword Generation for RabbitMQ ##############
+resource "random_password" "mq_pass" {
+  length           = 16
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+########## Rassword Generation for RDS ##############1
+resource "random_password" "rds_pass" {
+  length           = 16
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+########## Get RabbitMQ User from SSM ##############
+data "aws_ssm_parameter" "mq_user" {
+  name = "/sandbox/euc101/mq_user"

```

Comment:
Please add a counter with `rabbitmq_create`.  
Also, change hard coded path with `env_class` and `env_name`.